serde_codegen/src/lib.rs

```diff
@@ -36,51 +36,67 @@ include!(concat!(env!("OUT_DIR"), "/lib.rs"));
 include!("lib.rs.in");
 
 #[cfg(feature = "with-syntex")]
-pub fn expand<S, D>(src: S, dst: D) -> Result<(), syntex::Error>
```

Comment:
Sorry for the confusing diff - this is just splitting `expand` into `expand` and `expand_str` with common code in `syntex_registry`.